packages/keyring-eth-simple/src/simple-keyring.test.ts

```diff
@@ -21,8 +21,10 @@ import {
 import { add0x, Hex } from '@metamask/utils';
 import assert from 'assert';
 import { keccak256 } from 'ethereum-cryptography/keccak';
+// eslint-disable-next-line @typescript-eslint/naming-convention
 import OldEthereumTx from 'ethereumjs-tx';
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
 import SimpleKeyring from '.';
```

Comment:
Why is the import different here compared with the file `packages/keyring-eth-simple/src/sample.ts`?

```ts
import SimpleKeyring from './simple-keyring';
```

Reply:
I did change the one in `sample.ts` to match the original import path!